### DISCUSSION TITLE
🧩 FEATURE REQUEST:  Better "Back" functionality while browsing through notifications

### DISCUSSION THREAD

**Discussion by donangel** [dailydotdev/daily#1925] [Feature Requests]
### Concise Description

Ensure that the notifications list provides more value to browse all the new content.

### Description

I'm using the Notifications list to track what I've missed since my last visit. Today, when I click on a notification, it takes me to the article in question, which I can read, etc. However, when I'm done reading that article and I click "Back," I'm taken to the beginning of my notifications list (see the attached video). This makes it hard to remember where I was before clicking on the article to read, as I have to re-scroll over all notifications again.

As a User, when I click "Back" on my browser, I'd like to be positioned in the place where I was in the Notifications list. This will make my "Notifications list" experience better and more valuable.

https://github.com/user-attachments/assets/2c6622fa-18f3-4d66-b663-e5c7d4f09378

### Solution Proposed

As a User, when I click "Back" on my browser, I'd like to be positioned in the place where I was in the Notifications list. This will make my "Notifications list" experience better and more valuable.

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @donangel,

Thanks a lot for the suggestion :)

**Comment by defmans7** [comment]
Bump. 
This is the same behaviour in the main feed in the app.

Is this the indented behaviour maybe?

I expect my feed to not be reset (back to top) when I go "back" from reading an article or notification. I get frustrated after a few times of this. End up spending much less time browsing than I would if I wasn't getting a bit annoyed.

**Comment by idoshamun** [comment]
@defmans7 it's definitely not intended behavior, we need to fix it. Just takes time. Maybe someone from the community would love to contribute this fix

**Comment by defmans7** [comment]
I didn't realise it was only the android app that is affected. Would love to contribute but only have minimal experience with Android dev tools.

I wasn't aware theres a 100% viable pwa which works great. Installed that as a shortcut to my device and more than happy to run with that 🙂

Will definitely check out of I can contribute in another way though. Thanks for the suggestion.

**Comment by idoshamun** [comment]
Our android app is literally the PWA wrapped as native app. So PWA and native app are the same
